src/constant/chainConfig/arbitrum.ts

```diff
@@ -0,0 +1,6 @@
+import { ChainConfig } from "@/constant/chainConfig/types"
+
+export const arbitrumConfig: ChainConfig = {
```

Comment:
My original vision was to have nothing hardcoded in the app, but that seems less and less likely as time goes on, so I'm introducing this `ChainConfig` concept where we can put things like contract addresses, categorization mappings, logo URLs, etc. I think this will help us declutter constants scattered around the app and we will always know where to edit if something needs changing.

Reply:
I will adapt the concentrator when the time comes.